.github/workflows/_test_pax.yaml

```diff
@@ -13,6 +13,11 @@ on:
         description: Extra command line args to pass to test-pax.sh
         default: ""
         required: false
+      SUFFIX:
```

Comment:
I am (slowly) updating the workflows to use the more explicit [`ARTIFACT_NAME` convention](https://github.com/NVIDIA/JAX-Toolbox/blob/dc190365f21b459c0b6f13e3911e462991eef611/.github/workflows/_build_jax.yaml#L36). Could you please implement it accordingly?